docs/usage/agent-identity.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: default
+parent: Usage
+---
+
+# Agent identity
+
+Zincati agent tries to derive an unique identity for the machine it is running on, by introspecting the underlying OS and reading user configuration.
```

Comment:
```suggestion
Zincati agent tries to derive a unique identity for the machine it is running on by introspecting the underlying OS and reading user configuration.
```

Reply:
`a` versus `an` for words like `unique` is admittedly a difference between American English and British English.

Reply:
I think I changed this at least three time before committing. Next time I'll just flip a coin and save some time ;)